### ISSUE TITLE
[solid-js/html] `attr:foo="123"` in `html` template outputs as verbatim `attr:foo="123"`, inconsistent with JSX

### ISSUE THREAD

**Issue by trusktr** [solidjs/solid#1996]
### Describe the bug

This JSX code,

```js
function Example() {
	return <some-el attr:foo="123">inspect element</some-el>
}
```

outputs the attribute `foo="123"` to the DOM, as expected.

[JSX playground example](https://playground.solidjs.com/anonymous/4ea1522c-e8c7-434c-b0d5-5943325e58d2)

This `html` template,

```js
function Example() {
	return html`<some-el attr:foo="123">inspect element</some-el>`
}
```

outputs verbatim `attr:foo="123"` to the DOM.

[`html` playground example](https://playground.solidjs.com/anonymous/3f8870fa-ecb7-4222-b81c-abec2a56b6b3)

### Your Example Website or App

https://playground.solidjs.com/anonymous/3f8870fa-ecb7-4222-b81c-abec2a56b6b3

### Steps to Reproduce the Bug or Issue

see playground

### Expected behavior

Expect behavior to be consistent across JSX and `html`.

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by takeramagan** [comment]
It seems the html takes whatever you put into the dom.  Should we use the standard html here? in you case, remove the attr, use html`<some-el foo="123">inspect element</some-el>`?

**Comment by ryansolid** [comment]
Yeah this just was never added probably. HTML renderer was something I created a couple days back in 2018 (I think) and over time got updates because the runtime improved. We probably should look at adding this.

**Comment by ryansolid** [comment]
We won't be changing Solid's official Tag Template solution in V1 as that would be breaking. But 2.0 has a new rewritten system that addresses many of the issues. This issue has been confirmed working on the 2.0 branch. Since it will never get resolution in 1.x, I'm closing as fixed.

Thank you for reporting the issue and helping us make Solid better than ever.
